src/user.ts

```diff
@@ -0,0 +1,83 @@
+// const <name>: <Typ> = <Inhalt>
+const possibleTitles: ['Dr.', 'Prof.', null]=['Dr.', 'Prof.', null];
+export type Title = typeof possibleTitles;
```

Comment:
Adjust the type for `Title` and also adjust the "real" type of `possibleTitles` as discussed.